presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -161,7 +164,14 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                         new PushProjectionThroughExchange()));
 
         builder.add(
-                new DesugaringOptimizer(metadata, sqlParser), // Clean up all the sugar in expressions, e.g. AtTimeZone, must be run before all the other optimizers
+                new IterativeOptimizer(
```

Comment:
I would glue this iterative optimizer with the subsequent. Ideally, plan is to have just a single iterative optimizer.